components/i2c/scd30/definition.json

```diff
@@ -0,0 +1,21 @@
+{
+  "displayName": "SCD30",
+  "i2cAddresses": [ "0x61"],
+  "subcomponents": [
+    {
+      "displayName": "Temperature Sensor",
+      "sensorType": "ambient-temp",
+      "defaultPeriod": 900
+    },
+    {
+      "displayName": "Humidity Sensor",
+      "sensorType": "humidity",
+      "defaultPeriod": 900
+    },
+    {
+        "displayName": "eCO2 Sensor",
```

Comment:
not eCO2 - its true CO2!

Reply:
Ok - resolved in https://github.com/adafruit/Wippersnapper_Components/pull/13/commits/598eb64dd4bed797df1b602f7dbf97d846fd0f34